src/CodePen.js

```diff
@@ -0,0 +1,10 @@
+import { URL } from 'url';
+
+export const shouldTransform = string =>
+  !!new URL(string).pathname.match(/\/pen/);
+
+export const getHTML = string => {
```

Comment:
Just to be consistent with the rest of the code

```suggestion
export const getHTML = url => {
```